client/src/components/header/header.component.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import ErrorBoundary from '../common/errorBoundary/errorBoundary.component';
+import createMainHeader from './mainHeader/mainHeader.component';
+import createDescriptionHeader from './descriptionHeader/descriptionHeader.component';
+
+class Header extends React.Component {
```

Comment:
Consider turning this component into container with JSX child components instead of function calls in the render method.